### DISCUSSION TITLE
Draw on screen

### DISCUSSION THREAD

**Discussion by mhalano** [xournalpp/xournalpp#4028] [General]
There is a way to draw over the screen instead of a notebook page? There is this software  [Groomit-MPX](https://github.com/bk138/gromit-mpx) that do this, but I would prefer to use Xournal++ since it's more feature-complete.

**Comment by netlimpopo** [comment]
Openboard is able to provide the requested feature. On Windows there is pp-ink, which is a great screen annotation software. 
Xournal++ currently does not have his feature included. I am not sure whether this is on the high priority list. It would definitely be helpful, nonetheless I have the feeling priorities are more on pdf handling right now and the new version. Interesting Idea though.

**Comment by mikeperalta1** [comment]
Any chance this feature is being worked on now?

**Reply by rolandlo** [reply]
Currently nobody seems to be working on it.
